tfhe/src/core_crypto/backends/fft/private/math/fft/mod.rs

```diff
@@ -13,6 +13,8 @@ use crate::core_crypto::commons::utils::izip;
 use crate::core_crypto::prelude::PolynomialSize;
 use aligned_vec::{avec, ABox};
 use concrete_fft::c64;
+#[cfg(feature = "__wasm_api")]
+use concrete_fft::ordered::FftAlgo;
```

Comment:
these changes are not relevant anymore with ServerKey gone for public encrytpion